src/pages/Favorites.js

```diff
@@ -0,0 +1,36 @@
+import { useContext } from "react";
+import Card from "../components/Card";
+import { RecipeContext } from "../context/RecipeContext";
+import styled from "styled-components";
+
+export default function Favorites() {
+  const { recipes, setRecipes } = useContext(RecipeContext);
+
+  return (
+    <>
+      <h2>Your Favorites</h2>
+      <CardsContainer>
+        {recipes
+          .filter((recipe) => recipe.favorite)
```

Comment:
A name for a value that holds a `boolean` should start often with the pre `is`
like 

`isFavorite` 

With this you can understand that this might hold an boolean value. With just
`recipe.favorite` you don't know. It can be anything. Maybe the data of the favorite recipe. It is not wrong, how you named the property, but it might be hard to understand. ;-) 

Might considering changing the naming.